lib/js/components/PopOver/PopOver.consts.ts

```diff
@@ -6,6 +6,7 @@ export const POP_OVER_COLORS = {
 export const POP_OVER_TRIGGER_ACTIONS = {
 	CLICK: 'click',
 	HOVER: 'hover',
+	NONE: 'none',
```

Comment:
are you sure that's enough? If we set `none`, we still render the html of this popover. Is this a good approach? In a component such as lesson status, we will always render the content of the popover, which will be unnecessary for most users - because they only see this popover once during onboarding.
![Zrzut ekranu 2024-05-9 o 11 54 10](https://github.com/bethinkpl/design-system/assets/4587534/8bb051fd-4c75-4133-8199-b2de0e0a2cff)